### PR TITLE
Use `.get` to avoid `KeyError` on voice attributes

### PR DESCRIPTION
**Pull request by thevickypedia**
Upon updating to macOS Ventura 13.0, I identified that it lacks the key `VoiceAge` among its attributes. Instead of using `[]` to get the dict values, I switched it to `.get` which is also a more safe and pythonic way of getting dict values.

### CODE REVIEW

**Comment by TheMG95** [comment]
I also have the same issue with Ventura and your fix seems to work. Hope this gets merged soon.

**Review by IlyaYakushevskiy — APPROVED** [review]

**Comment by thevickypedia** [comment]
@IlyaYakushevskiy Thanks for the approval. I don't have write access to this repo, can you please approve this PR and create a release to build a new version in pypi. That will be a great help for people using macOS Ventura.

**Comment by IlyaYakushevskiy** [comment]
Hello! 
Don’t have much experience with GitHub, but I’ve merged branches, hope this is what I’m supposed to do

P.S. Thank you for your tool! 

Ilya

**Comment by thevickypedia** [comment]
The PR is still open, you may refer the [docs](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/incorporating-changes-from-a-pull-request/merging-a-pull-request#merging-a-pull-request) page regarding merging if you have push access to the repository.
This [section of the docs](https://docs.github.com/en/repositories/releasing-projects-on-github/managing-releases-in-a-repository#creating-a-release) talks about creating a release. Hope this clarifies.

**Comment by duck-dev-go** [comment]
When will this be merged?

**Comment by TheMG95** [comment]
Probably never seeing that the last commit was 1.5 years ago

**Comment by ausaf-a** [comment]
Would it work if I made a new PR off the latest codebase with this same change?

**Comment by thevickypedia** [comment]
I don't think a new PR is going to do the trick as the author has not been active. I have reached out to him via email 🤞🏻

**Comment by IlyaYakushevskiy** [comment]
Unfortunately, I don't have write access to this repo. 
 "any approvals from people with these permissions are denoted with a green check mark, and approvals from people without these permissions have a gray check mark."

**Comment by Cre5cend0** [comment]
Could someone with write access please merge this commit? It seems to solve the issue as expected.

**Comment by julianurregoar** [comment]
Waiting for this merge 👍🏽

**Comment by thevickypedia** [comment]
This PR is a much needed bug fix, but I couldn't get the owner of this repo to deploy it and it has been 4 months. So I created a new pypi project to make installation easy for others out there.

https://pypi.org/project/py3-tts/

> This is just a beta version. I plan on making some improvements to the code base, but leaving all the credits to the original creator.

**Comment by nateshmbhat** [comment]
hi @thevickypedia . thanks for your contributions , i've been pre-occupied with things. do u mind re-opening this PR so that i can merge it. ?

**Comment by nateshmbhat** [comment]
hi @thevickypedia . thanks for your contributions , i've been pre-occupied with things. do u mind re-opening this PR so that i can merge it. ?

**Comment by nateshmbhat** [comment]
these PR changes have been merged as part of another larger PR with more fixes from @willwade and published.

**Pull request closed without merging**
